### ISSUE TITLE
How do you determine the omega_0

### ISSUE THREAD

**Issue by zzw-zwzhang** [david-knigge/ccnn#4]
Thanks for the interesting work.

I want to know how do you determine the value of omega_0?

**Comment by dwromero** [comment]
Hi @zzw-zwzhang,

sorry for my late response. 

We treat omega_0 as a hyperparameter. In other words, we select it by trying out different values and checking the validation accuracy. 

Cheers,
David

**Comment by zzw-zwzhang** [comment]
Is its initial choice related to kernel size and channel number?
